test/models/text_message_test.rb

```diff
@@ -14,31 +14,20 @@ class TextMessageTest < ActiveSupport::TestCase
   end
 
   test 'processes later' do
-    Sms::ProcessingJob.expects(:perform_later).with(text_messages(:michael_checking_in))
-
-    text_messages(:michael_checking_in).process_later
-  end
-
-  test 'processes as check-in with member' do
-    TextMessage::AttendanceProcessor.expects(:process).with(text_messages(:michael_checking_in)).once
-
-    text_messages(:michael_checking_in).process
-  end
-
-  test 'processes as registration without member' do
-    TextMessage::RegistrationProcessor.expects(:process).with(text_messages(:rashid_checking_in)).once
-
-    text_messages(:rashid_checking_in).process
+    assert_enqueued_jobs 1 do
```

Comment:
When Rails 5 is released, we'll be able to scope this to the specific `TextMessageProcessingJob` we expect to be enqueued.